webapp/src/errorList.tsx

```diff
@@ -42,15 +42,25 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
                                 icon={`inverted chevron ${isCollapsed ? 'up' : 'down'}`}
                                 title={collapseTooltip} onClick={this.onCollapseClick} />
 
-        const errorListInnerClasses = errorsAvailable ? "errorListInner" : "errorListInner noErrorsMessage"
+        let errorListContent;
+        if (errorsAvailable) {
+            if (isCollapsed) {
+                errorListContent = <div className="summaryMessage" onClick={this.onCollapseClick}>
+                    {lf("Uh oh! You have {0} error(s)!", errors.length)}
+                </div>
+            } else {
+                errorListContent = (errors).map(e =>
+                    <div key={errorKey(e)}>{`${e.messageText} - (${e.line + 1}:${e.column + 1})`}</div>)
+            }
+        } else {
+            errorListContent = <div>{lf("Everything seems fine!")}</div>
+        }
+
+        const errorListInnerClasses = isCollapsed ? "errorListInner errorListSummary" : "errorListInner";
```

Comment:
We typically do `class1 ${condition ? “Class2” : “”}`

Reply:
I just changed it by removing the errorListInnerClasses variable and placed it directly, is this what you were meaning?